### ISSUE TITLE
Multiple Accessibility Themes Can Be Enabled Simultaneously

### ISSUE THREAD

**Issue by BamaCharanChhandogi** [processing/p5.js-website#576]
### Most appropriate sections of the p5.js website?

Reference

### What is your operating system?

Windows

### Actual Behavior

![image](https://github.com/user-attachments/assets/ea7b679b-a820-42c9-b3ae-22e10ca7ccf0)

Currently, users are able to select and enable multiple accessibility themes (e.g., Dark Mode, Monochrome Mode) at the same time. This behavior is not common, as usually only one theme is active at a time. However, when all themes are selected, they function without causing visual or functional conflicts.

### Expected Behavior

Only one accessibility theme should be active at a time. When selecting a new theme, the previously selected theme should automatically be deselected. Alternatively, if multiple themes are intentionally allowed, the interface should clearly communicate that users can activate more than one.

### Steps to reproduce

- Navigate to the any page

### Would you like to work on the issue?

yes

**Comment by limzykenneth** [comment]
Enabling multiple accessibility features are by design and it is reasonable to expect intersectional accessibility requirements. I'm not fully sure why this is an issue though, would you be able to elaborate more on what you mean?

**Comment by aakansha2304** [comment]
Could you please assign this issue to me? I want to work on it.
